src/applications/toe/helpers.jsx

```diff
@@ -142,21 +142,21 @@ export function prefillTransformer(pages, formData, metadata, state) {
     userAddressLine2 = profile?.addressLine2;
     userCity = profile?.city;
     userState = profile?.stateCode;
-    userPostalCode = profile?.zipCode;
+    userPostalCode = profile?.zipcode;
```

Comment:
Missed this before, but are you getting address information other than `zipcode` to populate in state.user.profile? That's the only piece of address information I'm see in the profile object. If so, I think we should remove this entire if block.